src/jenkins/common/Openenclave.groovy

```diff
@@ -124,13 +124,13 @@ def emailJobStatus(String status) {
  * Compile open-enclave on Windows platform, generate NuGet package out of it, 
  * install the generated NuGet package, and run samples tests against the installation.
  */
-def WinCompilePackageTest(String dirName, String buildType, String hasQuoteProvider, Integer timeoutSeconds, String lvi_mitigation = 'None', String lvi_mitigation_skip_tests = 'ON') {
+def WinCompilePackageTest(String dirName, String buildType, String hasQuoteProvider, Integer timeoutSeconds, String LVIMitigation = 'None', String LVIMitigationSkipTests = 'ON') {
```

Comment:
should this be `lviMitigation`? That would be consistent with .NET's guidance on camel case abberviations: https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN

Obviously this isn't .NET but it was the first reference I found